src/app/app/onboarding/_components/OnboardingForm.tsx

```
@@ -55,64 +80,69 @@ export function OnboardingForm({ user }: { user: OnboardingUser }) {
       phoneNumber: (value: string) => {
-        if (!/^\d{10}$/.test(value)) return "Invalid phone number (10 digits)";
+        if (!/^\d{10}$/.test(value))
+          return "Invalid phone number (10 digits)";
         return null;
       },
-      location: (value: string) => (value.length > 0 ? null : "Location is required"),
-      companyName: (value: string) => (value.length > 0 ? null : "Business name is required"),
-      facebook: (value: string) => {
-        if (value.length && !/^(http|https):\/\/(www.)?facebook.com\/[^ "]+$/.test(value)) return "Invalid Facebook URL";
-      },
+      location: (value: string) =>
```

ESLint

[error] Expected object keys to be in natural insensitive ascending order. 'location' should be before 'phoneNumber'.

(sort-keys-fix/sort-keys-fix)